remote.go

```diff
@@ -790,6 +793,29 @@ func (wd *remoteWD) Screenshot() ([]byte, error) {
 	return ioutil.ReadAll(decoder)
 }
 
+func (wd *remoteWD) Log(typ string) ([]LogMessage, error) {
+	url := wd.requestURL("/session/%s/log", wd.id)
+	params := map[string]string{
+		"type": typ,
+	}
```

Comment:
nit: Maybe just inline this into the Marshal call since it is not reused and relatively uninteresting?

data, err := json.Marshal(map[string]string{"type": typ})